Makefile.pre.in

```diff
@@ -439,9 +439,16 @@ DTRACE_DEPS = \
 # Rules
 
 # Default target
-all:		@DEF_MAKE_ALL_RULE@
+all:		check-clean-src @DEF_MAKE_ALL_RULE@
```

Comment:
Would it be possible to add this dependency to $(BUILD_PYTHON) instead, to fix more "make" commands?

Reply:
Yes, this is much better and would cover all cases I think.

BTW a '.PHONY' declaration is missing for check-clean-src.